src/lesson_1/Run.java

```diff
@@ -0,0 +1,174 @@
+package lesson_1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Run {
+    static Scanner keyboard = new Scanner(System.in);
+    static List<User> users = new ArrayList<>();
+    static List<CoworkingSpace> spaces = new ArrayList<>();
+    static List<Reservation> reservationList = new ArrayList<>();
+
+    public static void main(String[] args) {
+        User customer = new User("customer", "customer", false);
+        User admin = new User("admin", "admin", true);
+        users.add(customer);
+        users.add(admin);
+
+
+
+        System.out.println("Welcome to Coworking Space Reservation Application");
+        while (true) {
+            System.out.println("Choose one of the following options:");
+            System.out.println("1. Admin Login");
+            System.out.println("2. Customer Login");
+            System.out.println("3. Exit");
+            switch (keyboard.nextLine()) {
+                case "1" -> {
+                    System.out.println("Enter Login");
+                    String login = keyboard.nextLine();
+                    System.out.println("Enter Password");
+                    String password = keyboard.nextLine();
+                    User user = checkLogin(login, password);
+                    if (Objects.requireNonNull(user).isAdmin()) {
+                        adminMenu();
+                    } else {
+                        System.out.println("Invalid Login or Password");
+                    }
+                }
+                case "2" -> {
+                    System.out.println("Enter Login");
+                    String login = keyboard.nextLine();
+                    System.out.println("Enter Password");
+                    String password = keyboard.nextLine();
+                    User user = checkLogin(login, password);
+                    if (!Objects.requireNonNull(user).isAdmin()) {
+                        customerMenu(user);
+                    } else {
+                        System.out.println("Invalid Login or Password");
+                    }
+                }
+                default -> {
+                    return;
+                }
+            }
+        }
+    }
+
+    private static User checkLogin(String login, String password) {
+        for (User user : users) {
+            if (user.getLogin().equals(login) && user.getPassword().equals(password)) {
+                return user;
+            }
+        }
+        return null;
+    }
+
+    private static void adminMenu() {
+        while (true) {
```

Comment:
add some boolean flag before while with true value and make it false in default case